Documentation/building/linux-instructions.md

```diff
@@ -37,7 +40,7 @@ ellismg@linux:~$ sudo apt-get update
 
 Then install the packages you need:
 
-`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev`
+`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext libicu-dev curl libcurl4-openssl-dev libssl-dev`
```

Comment:
Hi, I had wget installed, but when running ./build.sh native (corefx) I hit an error that curl was not installed and had to add curl, libcurl4-openssl-dev and libssl-dev to get the script to run.